### PR TITLE
Update copyright information and author file

### PR DESCRIPTION
**Pull request by LukasReschke**
Probably nice for the people that contributed to 9.0 to see themselves in the AUTHORS file :)

### CODE REVIEW

File: apps/user_ldap/user_ldap.php [hunk 1]
```diff
@@ -9,6 +9,7 @@
  * @author Renaud Fortier <Renaud.Fortier@fsaa.ulaval.ca>
  * @author Robin Appelman <icewind@owncloud.com>
  * @author Robin McCorkell <robin@mccorkell.me.uk>
+ * @author Scrutinizer Auto-Fixer <auto-fixer@scrutinizer-ci.com>
```
Comment (nickvergessen) [inline apps/user_ldap/user_ldap.php]: fix it?!

**Comment by nickvergessen** [comment]
+ the duplicated maci entry,

looks good otherwise

File: apps/user_ldap/user_ldap.php (hunk 1 above)
Comment (LukasReschke) [inline apps/user_ldap/user_ldap.php]: On it…

**Comment by Xenopathic** [comment]
MVP: Scrutinizer :laughing:

**Comment by LukasReschke** [comment]
All adjusted.

**Pull request merged**